docs/02-for-app-authors/12-linter.md

```diff
@@ -76,7 +83,12 @@ Warnings from the linter are non-fatal but they should be resolved, if possible.
 | Warning                                           | Explanation                                                                                                |
 | --------------------------------------------------| -----------------------------------------------------------------------------------------------------------|
 | `appstream-missing-developer-name`                | Appdata file is missing `developer_name`.                                                                  |
+| `appstream-missing-project-license`               | Appdata file is missing `project_license`.                                                                 |
+| `appstream-name-too-long`                         | `name` in appdata file is greater than 20 characters.                                                      |
+| `appstream-summary-too-long`                      | `summary` in appdata file is greater than 35 characters.                                                   |
+| `appstream-screenshot-missing-caption`            | Appdata is either missing screenshot captions or has empty captions.                                       |
 | `appstream-screenshots-files-not-found-in-ostree` | Screenshot files missing from ostree ref.                                                                  |
+| `desktop-file-low-quality-category`               | Desktop file has no proper category and has some combination of `KDE, GTK, Qt, Application, GNOME` only.   |
```

Comment:
```suggestion
| `desktop-file-low-quality-category`               | Desktop file has no proper category or has some combination of `KDE, GTK, Qt, Application, GNOME` only.   |
```

Reply:
I can fix it once I get back.

Individual low quality categories won't trigger the warning. For example `Qt;Network` is fine because there will be at least one "good" category.

It'll only warn when there is no "good" category defined ie. some combination of those only and nothing else.

Reply:
I guess something like "Desktop file has some combination of only low quality categories (Qt, KDE,...)"